core/endorser/msgvalidation_test.go

```diff
@@ -446,31 +446,20 @@ var _ = Describe("Validate", func() {
 		})
 	})
 
-	Context("when the identity is actually not a validly serialized proto", func() {
-		BeforeEach(func() {
-			up.SignatureHeader.Creator = []byte("garbage")
-			up.ChannelHeader.TxId = "8f9de857052f103caee0fef35f66766562b4b4c2a14af34e9626351de52edfc4"
-		})
-
-		It("returns an auth error", func() {
-			err := up.Validate(fakeIdentityDeserializer)
-			Expect(err).To(MatchError("access denied: channel [channel-id] creator org unknown, creator is malformed"))
-		})
-	})
-
```

Comment:
Removed test, since removed redundant deserialization of the `up.SignatureHeader.Creator` and there is a similar test for `idDeserializer.DeserializeIdentity` which covers it, see: `msp/msp_test.go:207`